Framework/Core/Windows/Devices/Pwm/PwmController.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using Windows.Devices.Pwm.Provider;
+
+namespace Windows.Devices.Pwm
+{
+    public sealed class PwmController
+    {
+        private IPwmControllerProvider m_provider;
+        private double m_minFrequency;
+        private double m_maxFrequency;

```

Comment:
The main reason is that these shouldn't be allowed to change, even if the provider reports a different value in the future.  However, I can remove these cached values if you feel strongly.